### PR TITLE
Updated prerequisites + connecting to GDN

### PR DESCRIPTION
**Pull request by ghost**
- Copied formatting from 'streams' getting started guide

### CODE REVIEW

File: docs/collections/graphs/quickstart.md [hunk 1]
```diff
@@ -22,100 +22,65 @@ Edges have a direction, with their relations `_from` and `_to` pointing from one
 - INBOUND: `_from` ← `_to`
 - ANY: `_from` ↔ `_to`.
 
-## Example
+## Prerequisites
 
-For this example, assume the following credentials:
+- Access to a [Macrometa account](https://auth.paas.macrometa.io/) with sufficient permissions to create streams.
+- Have your API Key ready. For more information, refer to [Create API Keys](../account-management/api-keys/create-api-keys.md).
```
Comment (jamesm-macrometa) [inline docs/collections/graphs/quickstart.md]:
```suggestion
- An API key. For more information, refer to [Create API Keys](../account-management/api-keys/create-api-keys.md).
```

Minor change to make list grammatically consistent

File: docs/collections/graphs/quickstart.md [hunk 2]
```diff
@@ -22,100 +22,65 @@ Edges have a direction, with their relations `_from` and `_to` pointing from one
...
+- Install the appropriate SDK. For more information, refer to [Install SDKs](../sdks/install-sdks.md).
```
Comment (jamesm-macrometa) [inline docs/collections/graphs/quickstart.md]:
```suggestion
- Appropriate SDK installed. For more information, refer to [Install SDKs](../sdks/install-sdks.md).
```

Minor change to make list grammatically consistent

File: docs/collections/graphs/quickstart.md [hunk 3]
```diff
@@ -22,100 +22,65 @@ Edges have a direction, with their relations `_from` and `_to` pointing from one
...
 
-- Tenant name is `nemo@nautilus.com`.
-- User password is `xxxxxx`.
+### Step 1. Connect to GDN
```
Comment (jamesm-macrometa) [inline docs/collections/graphs/quickstart.md:31]: I would either remove "Step 1" or add "Step 2," "Step 3," etc. to the rest of the headers.

File: docs/collections/graphs/quickstart.md [hunk 4]
```diff
@@ -22,100 +22,65 @@ Edges have a direction, with their relations `_from` and `_to` pointing from one
...
 
-### SDK Download
+The first step in using the Macrometa Global Data Network (GDN) is to establish a connection to a local region. When this code executes, it initializes the server connection to the region URL you specified. For more information about connecting to the GDN, refer to [Authentication](../account-management/auth/index.md).
```
Comment (jamesm-macrometa) [inline docs/collections/graphs/quickstart.md]:
```suggestion
To use graphs with Macrometa Global Data Network (GDN), you must first establish a connection to a local region. When this code runs, it initializes the server connection to the region URL you specified. For more information about connecting to GDN, refer to [Authentication](../account-management/auth/index.md).
```

@diana-macrometa Have we had a discussion about referring to GDN as "GDN" vs "the GDN"?

**Review by jamesm-macrometa — APPROVED** [review]
Looks good!  Just a few minor suggestions.

File: docs/collections/graphs/quickstart.md (hunk 4 above)
Comment (diana-macrometa) [reply docs/collections/graphs/quickstart.md]: Not really. I would think it depends on context.

File: docs/collections/graphs/quickstart.md (hunk 3 above)
Comment (diana-macrometa) [reply docs/collections/graphs/quickstart.md:31]: Check the other quickstarts and getting started topics. I think this is consistent with them.

Also, we need to rewrite the whole thing anyway.

**Review by diana-macrometa — APPROVED** [review]

File: docs/collections/graphs/quickstart.md (hunk 3 above)
Comment (diana-macrometa) [reply docs/collections/graphs/quickstart.md:31]: Dallas, don't worry about the step numbers for now. James and I will deal with it when we test and rewrite this quickstart.

Comment (jamesm-macrometa) [reply docs/collections/graphs/quickstart.md:31]: Yeah, after posting my last comment I saw we have other Quickstarts similarly formatted and never edited my comment.  We can make it consistent when we rewrite.

**Comment by diana-macrometa** [comment]
@dallas-macrometa, will you be finishing this soon?

**Comment by diana-macrometa** [comment]
NVM, realized it's done!

**Pull request merged**